ikasaneip/platform/ikasan-eip-standalone-bom/pom.xml

```diff
@@ -1065,6 +1059,31 @@
             </dependency>
             <!-- end The SFTP library -->
 
+            <dependency>
+                <groupId>com.amazonaws</groupId>
+                <artifactId>aws-java-sdk-bom</artifactId>
+                <version>${version.aws.java.sdk}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+
+            <dependency>
+                <groupId>org.bouncycastle</groupId>
+                <artifactId>bcprov-jdk16</artifactId>
+                <version>${version.bouncycastle}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>javax.validation</groupId>
+                <artifactId>validation-api</artifactId>
+                <version>${version.javax.validation.api}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>org.glassfish</groupId>
+                <artifactId>javax.el</artifactId>
```

Comment:
Why would we need to add dependency on glassfish?

Reply:
This is used for the JSR 380 bean validation :- "JSR 380 supports variable interpolation, allowing expressions inside the violation messages. To parse these expressions, we'll add the javax.el dependency from GlassFish, that contains  an implementation of the Expression Language specification:" [source](https://www.baeldung.com/javax-validation)

Reply:
Checked this glassfish dependency pulls in no other dependencies - its also what spring uses itself.